File: Example/Objective-C/Paged Tabs Example/Screens/ChildViewController.m
//
//  ViewController.m
//  TabbedPageViewController
//
//  Created by Merrick Sapsford on 24/12/2015.
//  Copyright © 2015 Merrick Sapsford. All rights reserved.
//

#import "ChildViewController.h"

@implementation ChildViewController

@synthesize tabBarView;

- (void)viewDidLoad {
    [super viewDidLoad];
    self.titleLabel.text = [NSString stringWithFormat:@"Page %i", (int)(self.pageIndex + 1)];
}

@end

File: Example/Swift/Paged Tabs Example/Source/UI/Screens/ChildViewController.swift
//
//  ChildViewController.swift
//  Paged Tabs Example
//
//  Created by Merrick Sapsford on 04/05/2016.
//  Copyright © 2016 Merrick Sapsford. All rights reserved.
//

import UIKit
import MSSTabbedPageViewController

class ChildViewController: UIViewController {
    
    // MARK: Vars
    
    @IBOutlet weak var titleLabel: UILabel?
    
    // MARK: Lifecycle
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.titleLabel?.text = String(format: "Page %d", self.pageIndex + 1)
    }
}
